.circleci/config.yml

```diff
@@ -252,7 +256,9 @@ jobs:
           command: rmdir ~/calva && cp -r /tmp/build ~/calva
       - run:
           name: Apt install missing dependencies
-          command: sudo apt update && sudo apt install -y libnss3
+          command: |
+            sudo apt-key adv --keyserver keyserver.ubuntu.com --recv-keys E88979FB9B30ACF2
```

Comment:
What's up with this change? Just curious

Reply:
Should have put a comment there =)

Suddenly the integration/e2e test runs bailed because some google chrome related docker image is signed in some no-longer-supported way. The change basically registers the key as trusted.

Here's a the failing run that triggered the change: https://app.circleci.com/pipelines/github/BetterThanTomorrow/calva/7301/workflows/cde8710c-bb6a-46e7-b1a8-b4b142408824/jobs/37763